data/formatters/generic.yaml

```diff
@@ -462,6 +462,24 @@ short_message:
 - '{file_md5}'
 ---
 type: 'conditional'
+data_type: 'nids:alert:fast'
+message:
+- 'Message: {message}'
+- 'Source IP: {source_ip}'
+- 'Source Port: {source_port}'
+- 'Destination IP: {destination_ip}'
+- 'Destination Port: {destination_port}'
+- 'Protocol: {protocol}'
+- 'Classification: {classifciation}'
+- 'Rule ID: {rule_id}'
```

Comment:
nit: per style guide don't use abbreviations

Reply:
Changed in 654138576ceba8f8a7eff21dbbe4c8d2b9b31c5d.